### PR TITLE
fix panic: send on closed channel on windows

### PR DESCRIPTION
**Pull request by azr**
if the term is resized when `ReadString` is being called and no one is reading form ws, we get a `panic: send on closed channel`.

It could make sense to poll getconsolescreenbufferinfo calls instead of this:

something started from the `sigwinch()` func

### CODE REVIEW

File: tty_windows.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"syscall"
 	"unsafe"
 
-	"github.com/mattn/go-isatty"
```
Comment (mattn) [inline tty_windows.go]: do not change

**Comment by mattn** [comment]
I don't understand what you want to fix (as far as I can see your change of code)

File: tty_windows.go (hunk 1 above)
Comment (azr) [reply tty_windows.go]: ah, go-imports fixed this for me ! 🙂

**Comment by azr** [comment]
Hey @mattn, here's a sample crashing code, just start it and resize the windows.

``` golang
package main

import (
	"log"

	"github.com/mattn/go-tty"
)

func main() {
	tty, err := tty.Open()
	if err != nil {
		log.Fatal(err)
	}

	str, err := tty.ReadString()
	log.Printf("str: %s, err: %v", str, err)
	tty.Close()
}
```

**Comment by mattn** [comment]
I prefer to use context.Context instead of closeC.

**Comment by mattn** [comment]
Looks good to me. Thank you.

**Pull request merged**